app-config-dev.yaml

```diff
@@ -0,0 +1,68 @@
+app:
+  title: Backstage App
+  baseUrl: http://localhost:3000
+
+organization:
+  name: AppEng-Backstage
+
+backend:
+  # Used for enabling authentication, secret is shared by all backend plugins
+  # See https://backstage.io/docs/auth/service-to-service-auth for
+  # information on the format
+  # auth:
+  #   keys:
+  #     - secret: ${BACKEND_SECRET}
+  baseUrl: http://localhost:7007
+  listen:
+    port: 7007
+    # Uncomment the following host directive to bind to specific interfaces
+    # host: 127.0.0.1
+  csp:
+    connect-src: ["'self'", 'http:', 'https:']
+    # Content-Security-Policy directives follow the Helmet format: https://helmetjs.github.io/#reference
+    # Default Helmet Content-Security-Policy values can be removed by setting the key to false
+  cors:
+    origin: http://localhost:3000
+    methods: [GET, HEAD, PATCH, POST, PUT, DELETE]
+    credentials: true
+  database:
```

Comment:
for the dev i prefer in memory database. just my preference. You can check with Sandeep also if he feels diferent.

Reply:
This file is used for OpenShift deployment. I am installing the PostgreSQL database within template. I can rename the file to **app-config-openshift.yaml** if that makes it clearer. For local development, **app-config.yaml** will be used, which is already set up with an in-memory database

Reply:
I think renaming makes much more sense and is clear. Thank you.